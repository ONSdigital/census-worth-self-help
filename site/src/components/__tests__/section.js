import React from "react"
import renderer from "react-test-renderer"

import Section from "../section"

describe("Section", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Section>children</Section>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})