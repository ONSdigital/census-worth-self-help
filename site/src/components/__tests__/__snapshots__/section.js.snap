// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section renders correctly 1`] = `
<div
  css={
    Object {
      "map": undefined,
      "name": "ypl6yy",
      "next": undefined,
      "styles": "
        border-bottom: 1px rgb(229, 229, 234) solid;
        padding: 15px 0px;
      ",
    }
  }
>
  children
</div>
`;
